packages/angular_devkit/core/node/host.ts

```diff
@@ -265,24 +265,21 @@ export class NodeJsSyncHost implements virtualFs.Host<fs.Stats> {
     return this.isDirectory(path).pipe(
       concatMap(isDir => {
         if (isDir) {
-          // Since this is synchronous, we can recurse and safely ignore the result.
-          for (const name of fs.readdirSync(getSystemPath(path))) {
-            this.delete(join(path, name)).subscribe();
-          }
-          try {
+          const dirPaths = fs.readdirSync(getSystemPath(path));
+          const rmDirComplete = new Observable((obs) => {
             fs.rmdirSync(getSystemPath(path));
```

Comment:
Unless the minimum version of rxjs is increased everywhere, a try/catch block is needed to work around a rxjs defect (https://github.com/ReactiveX/rxjs/issues/3740).

Reply:
Will re-add with the comment that the rest in https://github.com/angular/angular-cli/commit/cfa03826facc37bb8911eb584eafad856c13ce8a have.